xtable-hudi/pom.xml

```diff
@@ -0,0 +1,277 @@
+                                    <include>org.apache.parquet:parquet-column</include>
+                                    <include>org.apache.parquet:parquet-avro</include>
+                                    <include>org.apache.parquet:parquet-common</include>
+                                    <include>org.apache.parquet:parquet-encoding</include>
+                                    <include>org.apache.parquet:parquet-hadoop</include>
+                                    <include>org.apache.parquet:parquet-format-structures</include>
+                                    <include>org.apache.hudi:hudi-java-client</include>
+                                    <include>org.apache.hudi:hudi-client-common</include>
+                                    <include>org.apache.hudi:hudi-common</include>
+                                    <include>org.apache.avro:avro</include>
+                                    <include>com.google.guava:guava</include>
+                                    <!-- hbase -->
+                                    <include>org.apache.hbase:hbase-client</include>
+                                    <include>org.apache.hbase:hbase-common</include>
+                                    <include>org.apache.hbase:hbase-hadoop-compat</include>
+                                    <include>org.apache.hbase:hbase-metrics</include>
+                                    <include>org.apache.hbase:hbase-metrics-api</include>
+                                    <include>org.apache.hbase:hbase-protocol</include>
+                                    <include>org.apache.hbase:hbase-protocol-shaded</include>
+                                    <include>org.apache.hbase:hbase-server</include>
+                                    <include>org.apache.hbase.thirdparty:hbase-shaded-miscellaneous</include>
+                                    <include>org.apache.hbase.thirdparty:hbase-shaded-netty</include>
+                                    <include>org.apache.hbase.thirdparty:hbase-shaded-protobuf</include>
+                                </includes>
+                            </artifactSet>
+                            <relocations combine.children="append">
+                                <relocation>
+                                    <pattern>org.apache.parquet.</pattern>
+                                    <shadedPattern>org.apache.xtable.shade.org.apache.parquet.</shadedPattern>
+                                </relocation>
+                                <relocation>
+                                    <pattern>org.apache.hudi.</pattern>
+                                    <shadedPattern>org.apache.xtable.shade.org.apache.hudi.</shadedPattern>
+                                    <excludes>
+                                        <exclude>org.apache.hudi.io.storage.HoodieHBaseKVComparator</exclude>
+                                    </excludes>
```

Comment:
One way to validate the conjecture above is to remove this exclusion to see if the test passes or a different error is thrown (by removing the exclusion, there should be no clash any more, and the `org.apache.xtable.shade.org.apache.hudi.io.storage.HoodieHBaseKVComparator` is written as the comparator in the HFile if the test table is prepared using the XTable bundle).  Still, we need to figure out how to properly handle `org.apache.hudi.io.storage.HoodieHBaseKVComparator` as the comparator class since the fully-qualified class name needs to be preserved for reading HFile format in Hudi.